frontend/pages/hosts/details/cards/Software/SoftwareTableConfig.tsx

```diff
@@ -99,6 +99,22 @@ const condenseVulnerabilities = (vulns: string[]): string[] => {
     : condensed;
 };
 
+const withBundleTooltip = (name: string, bundle: string) => (
```

Comment:
It shouldIt should be `renderBundleTooltip` to match the naming convention used elsewhere.

Reply:
Ooo nice catch. I'll fix it from the naming on `software/ManageSoftwarePage/SoftwareTableConfig.tsx` I took `withBundleTooltip` from.